packages/llamaindex/src/cloud/LLamaCloudFileService.ts

```diff
@@ -0,0 +1,171 @@
+import {
+  FilesService,
+  PipelinesService,
+  ProjectsService,
+} from "@llamaindex/cloud/api";
+import type { Metadata, NodeWithScore } from "@llamaindex/core/schema";
+import type { CloudConstructorParams } from "./constants.js";
+import { initService } from "./utils.js";
+
+const OUTPUT_DIR = "output/llamacloud";
+const FILE_DELIMITER = "$";
+
+export type LLamaCloudFileServiceConfigs = {
+  outputDir: string;
+  fileDelimiter: string;
+};
+
+export class LLamaCloudFileService {
+  params?: CloudConstructorParams;
+  configs: LLamaCloudFileServiceConfigs;
+
+  constructor(
```

Comment:
Can probably be removed after the changes below. Then the call would be static like PipelineService